src/pkg/credentials/corso.go

```diff
@@ -4,23 +4,29 @@ import (
 	"os"
 
 	"github.com/alcionai/clues"
+
+	"github.com/alcionai/corso/src/internal/common/str"
 )
 
 // envvar consts
 const (
 	CorsoPassphrase = "CORSO_PASSPHRASE"
+	// Corso Flags
+	CorsoPassphraseFN = "passphrase"
```

Comment:
Anything related to flags is purely the concern of the CLI, and should not appear in other layers.  This value, the FV below it should be moved to `cli/utils.go`.  That value can then be passed into this func.

Reply:
I tried that. But getting stuck due to `cyclic dependency`. credentials and cli seems to be quite dependent packages. Also, out of curiosity - why do we have two packages- `options` and `utils packages' flag.go` handling the flags?

Reply:
> But getting stuck due to cyclic dependency. credentials and cli seems to have quite dependent packages.

Well, that should only happen if credentials is importing the CLI, yeah?  Which isn't an import direction we should be supporting anyway.  Higher level layers such as apis (cli, sdk) should import from lower levels (storage, credentials), but not the other way around.

I'd try to track down how the cycle is happening first and see if you can resolve that more directly; potentially by passing flag values in through a func parameter instead of referencing the global instance?  Distributing flag ownership into different packages as a solution to circular imports is not the right way to go about it, since that only injects a workaround, and doesn't solve the underlying cause.

> Also, out of curiosity - why do we have two packages- options and utils packages' flag.go handling the flags?

Yeah, I agree, it's a bit odd.  Feel free to write up a tech debt ticket for something like moving options into utils and we can refactor when we have a moment.

Reply:
With some debugging - The import cycle is between cli's config and cli's util package. 
`import cycle not allowed: import stack: [github.com/alcionai/corso/src/cli/restore github.com/alcionai/corso/src/cli/repo github.com/alcionai/corso/src/cli/config github.com/alcionai/corso/src/cli/utils github.com/alcionai/corso/src/cli/config]`

The thing is - its not a single `cli` package. There are multiple small packages like - config, utils inside it. And which is correct for code structuring. But then there should be a fix route like -  config can import utils or vice-versa.